podman/tests/integration/test_images.py

```diff
@@ -144,6 +145,19 @@ def test_build(self):
         self.assertIsNotNone(image)
         self.assertIsNotNone(image.id)
 
+    def test_build_cache(self):
```

Comment:
more tests are probably needed to ensure that the defaults are passed, and images.build also accepts the other parameters